lib/rust/web/src/lib.rs

```diff
@@ -721,6 +721,83 @@ pub fn ignore_context_menu(target: &EventTarget) -> EventListenerHandle {
 // === Event Listeners ===
 // =======================
 
+// === EventListenerHandleOptions ===
+
+/// Structure representing event listener options used by [`EventListenerHandle`].
+///
+/// The handle cannot just use [`AddEventListenerOptions`], as it needs to construct also
+/// [`EventListenerOptions`] for removing the listener on drop, and values cannot be read from the
+/// former easily.
+///
+/// Description of fields is cited from the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#parameters)
+#[derive(Copy, Clone, Debug, Default)]
+pub struct EventListenerHandleOptions {
+    /// From
+    /// > A boolean value that, if true, indicates that the function specified by listener will
+    /// > never call preventDefault()
+    ///
+    /// `None` means a default value, which in turn may depend on the event kind.
+    pub passive: Option<bool>,
+    /// > A boolean value indicating that events of this type will be dispatched to the registered
+    /// > listener before being dispatched to any EventTarget beneath it in the DOM tree. If not
+    /// > specified, defaults to false.
+    pub capture: bool,
+    /// > A boolean value indicating that the listener should be invoked at most once after being
+    /// > added. If true, the listener would be automatically removed when invoked. If not
+    /// > specified, defaults to false.
+    pub once:    bool,
+}
+
+#[allow(missing_docs)]
+impl EventListenerHandleOptions {
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    pub fn passive(mut self) -> Self {
+        self.passive = Some(true);
+        self
+    }
+
+    pub fn not_passive(mut self) -> Self {
+        self.passive = Some(false);
+        self
+    }
+
```

Comment:
Please add here docs. It's not obvious what these do after reading the name if someone (like me) do not remember what the JS side does.